Sources/Charcoal/Filters/FreeText/FreeTextFilterViewController.swift

```diff
@@ -53,34 +54,71 @@ public class FreeTextFilterViewController: UIViewController {
 
     private lazy var tableView: UITableView = {
         let tableView = UITableView(frame: .zero, style: .plain)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.register(IconTitleTableViewCell.self)
+        tableView.removeLastCellSeparator()
         tableView.dataSource = self
         tableView.delegate = self
-        tableView.register(IconTitleTableViewCell.self)
-        tableView.tableFooterView = UIView()
-        tableView.translatesAutoresizingMaskIntoConstraints = false
         return tableView
     }()
 
-    init(filter: Filter, selectionStore: FilterSelectionStore) {
+    // MARK: - Init
+
+    init(filter: Filter, selectionStore: FilterSelectionStore, notificationCenter: NotificationCenter = .default) {
         self.filter = filter
         self.selectionStore = selectionStore
+        self.notificationCenter = notificationCenter
         super.init(nibName: nil, bundle: nil)
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    // MARK: - Lifecycle
+
+    public override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        let notificationCenter = NotificationCenter.default
```

Comment:
I guess you could remove this line, since you get a `NotificationCenter` through the constructor.

Reply:
Nice catch!